### PR TITLE
Blood: Add crouch toggle

### PR DESCRIPTION
**Pull request by tmyqlfpir**
This PR adds crouch toggle functionality, and resolves issue #781

### CODE REVIEW

**Comment by Hendricks266** [comment]
Instead of a modifier on the behavior of the crouch key, I would prefer Crouch and Toggle Crouch be separate and coexisting gamefuncs like how I implemented it in EDuke32.

File: source/duke3d/src/menus.cpp [hunk 1]
```diff
@@ -914,7 +914,7 @@ static MenuEntry_t ME_MOUSESETUP_MOUSEAIMING = MAKE_MENUENTRY( "Vertical aiming:
 static MenuOption_t MEO_MOUSESETUP_INVERT = MAKE_MENUOPTION( &MF_Redfont, &MEOS_YesNo, &ud.mouseflip );
 static MenuEntry_t ME_MOUSESETUP_INVERT = MAKE_MENUENTRY( "Inverted aiming:", &MF_Redfont, &MEF_BigOptionsRtSections, &MEO_MOUSESETUP_INVERT, Option );
 
-static MenuRangeFloat_t MEO_MOUSESETUP_XSENSITIVITY = MAKE_MENURANGE( &CONTROL_MouseAxesSensitivity[0], &MF_Redfont, 1.f, 50.f, 50.f, 99, DisplayTypeInteger|EnforceIntervals );
+static MenuRangeFloat_t MEO_MOUSESETUP_XSENSITIVITY = MAKE_MENURANGE( &CONTROL_MouseAxesSensitivity[0], &MF_Redfont, 0.f, 50.f, 50.f, 99, DisplayTypeInteger|EnforceIntervals );
```
Comment (nukeykt) [inline source/duke3d/src/menus.cpp:917]: position count should be adjusted to 101

File: source/rr/src/menus.cpp [hunk 2]
```diff
@@ -1018,10 +1018,10 @@ static MenuEntry_t ME_MOUSESETUP_MOUSEAIMING = MAKE_MENUENTRY( "Vertical aiming:
 static MenuOption_t MEO_MOUSESETUP_INVERT = MAKE_MENUOPTION( &MF_Redfont, &MEOS_YesNo, &ud.mouseflip );
 static MenuEntry_t ME_MOUSESETUP_INVERT = MAKE_MENUENTRY( "Invert aiming:", &MF_Redfont, &MEF_BigOptionsRt, &MEO_MOUSESETUP_INVERT, Option );
 
-static MenuRangeFloat_t MEO_MOUSESETUP_XSENSITIVITY = MAKE_MENURANGE( &CONTROL_MouseAxesSensitivity[0], &MF_Redfont, 1.f, 10.f, 10.f, 91, DisplayTypeInteger|EnforceIntervals );
+static MenuRangeFloat_t MEO_MOUSESETUP_XSENSITIVITY = MAKE_MENURANGE( &CONTROL_MouseAxesSensitivity[0], &MF_Redfont, 0.f, 10.f, 10.f, 91, DisplayTypeInteger|EnforceIntervals );
```
Comment (nukeykt) [inline source/rr/src/menus.cpp:1021]: same

File: source/rr/src/menus.cpp [hunk 3]
```diff
@@ -1018,10 +1018,10 @@ static MenuEntry_t ME_MOUSESETUP_MOUSEAIMING = MAKE_MENUENTRY( "Vertical aiming:
...
 static MenuEntry_t ME_MOUSESETUP_HORIZONTALSENSITIVITY = MAKE_MENUENTRY( "Horiz sens.:", &MF_Redfont, &MEF_BigOptionsRtSections, &MEO_MOUSESETUP_XSENSITIVITY, RangeFloat );
 
-static MenuRangeFloat_t MEO_MOUSESETUP_YSENSITIVITY = MAKE_MENURANGE( &CONTROL_MouseAxesSensitivity[1], &MF_Redfont, 1.f, 10.f, 10.f, 91, DisplayTypeInteger|EnforceIntervals );
+static MenuRangeFloat_t MEO_MOUSESETUP_YSENSITIVITY = MAKE_MENURANGE( &CONTROL_MouseAxesSensitivity[1], &MF_Redfont, 0.f, 10.f, 10.f, 91, DisplayTypeInteger|EnforceIntervals );
```
Comment (nukeykt) [inline source/rr/src/menus.cpp:1024]: here too

**Review by nukeykt — CHANGES_REQUESTED** [review]

**Review by nukeykt — APPROVED** [review]

**Pull request merged**